### PR TITLE
Added Employee module using typeorm

### PR DESCRIPTION
**Pull request by stevespur**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -5,6 +5,12 @@
     "no-unused-expression": true
   },
   "rules": {
+    "variable-name": [
+      true,
+      "ban-keywords",
+      "check-format",
+      "allow-leading-underscore"
```
Comment (maybenull) [inline tslint.json]: no?

**Comment by maybenull** [comment]
whats the ispec and spec difference?

File: tslint.json (hunk 1 above)
Comment (stevespur) [reply tslint.json]: https://stackoverflow.com/questions/45167199/why-are-underscores-added-to-fields-of-a-typescript-class-in-angular4
https://github.com/angular/angular/issues/26393

and apparently the underscore is the default behavior in VSC and idea IDEs

**Comment by stevespur** [comment]
spec is unit, ispec is integration? still a bit unsure about this, it seems their e2e tests are what we call it tests

File: tslint.json (hunk 1 above)
Comment (maybenull) [reply tslint.json]: so its only required if you want a property names world and a method named world()?

**Comment by maybenull** [comment]
ya i think the e2e is an integration test and spec is just unit.

File: tslint.json (hunk 1 above)
Comment (stevespur) [reply tslint.json]: yea. the IDEs dont follow the getFoo convention, some people saying thats how Java does it, not Javascript

**Comment by stevespur** [comment]
I don't understand the two locations tho, why not have them all under test or all next to the files?  But yea, I was thinking of E2E as selemium type tests

**Pull request merged**